Tests/Unit/APIClientTests.swift

```diff
@@ -0,0 +1,128 @@
+import Quick
+import Nimble
+@testable import RRemoteConfig
+
+class APIClientSpec: QuickSpec {
+    struct TestStruct: Decodable, Equatable {
+        let foo: String
+    }
+    override func spec() {
+        describe("send function") {
+            class SessionMock: SessionProtocol {
```

Comment:
Is there a Swift mocking library that can make this easier? It seems like it could get really time consuming to manually create mocks.

Reply:
Yep, very time consuming. The language doesn't support it. See this post https://qualitycoding.org/swift-mocking/

I saw https://github.com/Brightify/Cuckoo which reads your source files and generates mocks but didn't try it yet. It seemed to have lots of unresolved open issues.

Reply:
I think it's worth giving Cuckoo a try at some point, but probably manual mocks are okay for now.